tot/socket/socket.conner.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 
 	errs "github.com/bdlm/errors"
-	log "github.com/sirupsen/logrus"
+	"github.com/bdlm/log"
```

Comment:
could not import github.com/bdlm/log (cannot find package "github.com/bdlm/log" in any of:
	/usr/local/go/src/github.com/bdlm/log (from $GOROOT)
	/tmp/tmp.4WnmxJoFcK/src/github.com/bdlm/log (from $GOPATH))